### PR TITLE
Added some missing enum constants.

### PR DESCRIPTION
**Pull request by SilverPhoenix99**

### CODE REVIEW

**Review by ioquatix — APPROVED** [review]

**Comment by cfis** [comment]
I didn't see this PR before I also updated `enum :kind`, but it would be good to merge in the other enum updates.

**Comment by ioquatix** [comment]
@cfis should I merge this change?

**Comment by cfis** [comment]
Yes I think so (note I have not verified it is correct, just assuming it is).

File: lib/ffi/clang/lib/type.rb [hunk 1]
```diff
@@ -108,7 +174,7 @@ class CXType < FFI::Struct
 			attach_function :get_pointee_type, :clang_getPointeeType, [CXType.by_value], CXType.by_value
 			attach_function :get_result_type, :clang_getResultType, [CXType.by_value], CXType.by_value
 			attach_function :get_canonical_type, :clang_getCanonicalType, [CXType.by_value], CXType.by_value
-			
+
```
Comment (ioquatix) [inline lib/ffi/clang/lib/type.rb:177]:
```suggestion
			
```

**Comment by ioquatix** [comment]
Most of this was already done, so manually pulled a few of the missing changes in <https://github.com/ioquatix/ffi-clang/commit/4c460e7cc24366f8ba37e372a6b52a1b207962a1>. Thanks for your contribution.

**Pull request closed without merging**